app/views/layouts/application.html.erb

```diff
@@ -19,5 +25,11 @@
 
     <%= render 'partials/navbar' %>
     <%= yield %>
+    <script>
+    $(document).on('click', '[data-toggle="lightbox"]', function(event) {
+                event.preventDefault();
+                $(this).ekkoLightbox();
```

Comment:
Am I wrong or are you doing this twice?

Reply:
you're wrong

Reply:
Please take a look at the coffee file added to this PR.